python/Ganga/GPIDev/Adapters/IGangaFile.py

```diff
@@ -71,6 +72,15 @@ def put(self):
         """
         raise NotImplementedError
 
+    def copyTo(self, targetPath):
+        """
+        Copy a the file to the local storage using the get mechanism
+        Args:
+            targetPath (str): Target path where the file is to copied to
+        """
+        self.localDir = targetPath
```

Comment:
Do we really want to be resetting the value of `localDir` here? Surely this function should be the one doing an arbitrary copy from some canonical location to the `teargetPath` and then `get()` should us this function to do its job?